public/static/js/admin.js

```diff
@@ -131,6 +155,13 @@ mciModule.controller('AdminSettingsController', ['$scope', '$window', '$http', '
       $scope.Settings.expansions = {};
     }
 
+    // Support transition to region-based EC2Keys struct -- TO BE DELETED
+    $scope.Settings.providers.aws.aws_id = $scope.tempDefaultEC2Key;
+    $scope.Settings.providers.aws.aws_secret = $scope.tempDefaultEC2Secret;
+
+    $scope.Settings.providers.aws.ec2_keys = [
```

Comment:
Are we currently not saving to the ec2_keys list?

Reply:
You're right, good catch -- I changed this to help clean up the messy data I created in staging.